docs/content/troubleshooting/csi_ug_troubleshooting_node_crash.md

```diff
@@ -45,32 +45,32 @@ Follow the following procedure to recover from a crashed node (see a [full examp
     kubectl delete pod <pod name> --grace-period=0 --force
     ```
 
-5.  Verify that the pod is now in a _Running_ state and that the pod has moved to worker-node1.
+5.  Verify that the pod is now in a _Running_ state and that the pod has moved to a _Ready_ node.
 
-<a name="full_example">For example:</a>
+    <a name="full_example">For example:</a>
```

Comment:
this appears to be under step 5, even though it sums up all the steps

Reply:
DONE